sets/technic/42124-off-road-buggy/powered-up-remote/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Powered Up Remote Control"
+maintainer:
+    user: "JorgePe"
+    name: "Jorge Pereira"
+image:
+    local: "powered-up-remote-buggy.jpg"
+description:
+    "Control LEGO Technic vehicles with the Powered Up Remote."
+video:
+    youtube: st_lcMgz618
+building_instructions:
+    external: https://www.lego.com/cdn/product-assets/product.bi.core.pdf/6351188.pdf
+code: "#program"
+---
+
+
+# Program
+
+This program expands the [basic driving program](../driving) with the Powered
+Up Remote.
```

Comment:
Is there a bit more we could say about the model or what you can do with it?

Reply:
Where do I put that?  Under "Description:" ?

"A simple off-road buggy with front and back suspension great for driving on rough terrain with a powerful Hub that allows you to add extra features later"

Reply:
I mean right here where I commented. It can be brief or a as long as you want. You can even write a few paragraphs (see the keyboard-remote example.)

Everything after the first `------` block is just regular markdown, with no special requirements. Another example is [this one](https://pybricks.com/projects/sets/boost/creative-toolbox/auto-builder/) with a section called "Further exploration" for other activities that the user can try.